### ISSUE TITLE
[weekly reports] Add summary also to daily results

### ISSUE THREAD

**Issue by rvykydal** [rhinstaller/kickstart-tests#704]
Quoting:

2) I think it could be useful to have such summary also for each day/scenario, eg
```
03/16/2022
    logs-daily-iso (SUMMARY):
```
It can became very useful for quick picture when there are mass failures, workflow timeouts, infra issues etc.

_Originally posted by @rvykydal in https://github.com/rhinstaller/kickstart-tests/pull/692#pullrequestreview-920265479_

**Comment by bcl** [comment]
By summary do you mean the pass/fail count for each day?

eg.

```
  03/31/2022
    logs-daily-iso (Ran 247, 245 passed, 2 failed):
        bindtomac-bridge-httpks
        clearpart-2
        clearpart-2
    logs-rhel8 (Ran 194, 192 passed, 2 failed):
        network-device-mac
        ntp-without-chrony
```

**Comment by rvykydal** [comment]
Yes.

**Comment by bcl** [comment]
Fixed by #705 I guess t doesn't know about lists? :)

**Comment by rvykydal** [comment]
Thank you!
